app/src/common-logging/common-logging-transformer.js

```diff
@@ -0,0 +1,67 @@
+const Constants = require('./common-logging-constants');
+
+class CommonLoggingTransformer {
+  constructor(options) {
+    this._defaults = {
+      env: options && options.env ? options.env : Constants.TRANSFORM_DEFAULT_ENV,
+      level: options && options.level ? options.level : Constants.TRANSFORM_DEFAULT_LEVEL,
+      pattern: options && options.pattern ? options.pattern : '',
+      retention: options && options.retention ? options.retention : Constants.TRANSFORM_DEFAULT_RETENTION
+    };
+
+    this._metadata = options && options.metadata && options.metadata === Object(options.metadata) ? options.metadata : undefined;
+    this._data = options && options.data && options.data === Object(options.data) ? options.data : undefined;
+  }
+
+  xform(message, options) {
+    if (!message) {
+      return;
+    }
+    let opts = options || {};
```

Comment:
Doesn't look like opts is reassigned - probably could be declared as a const instead

Reply:
got it